modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -318,10 +318,24 @@ private void onTableCreateInternal(ConfigurationNotificationEvent<TableView> ctx
                                         return CompletableFuture.completedFuture(new NodeStoppingException());
                                     }
                                     try {
-                                        // Multi invoke supposed to be here
-                                        metaStorageMgr.put(new ByteArray("some key"), null);
+                                        String tableName = replicasCtx.name(TableConfiguration.class);
+                                        TableConfiguration tableCfg = replicasCtx.config(TableConfiguration.class);
+
+                                        int partCount = tableCfg.partitions().value();
+
+                                        int newReplicas = replicasCtx.newValue();
+
+                                        for (int i = 0; i < partCount; i++) {
+                                            String partId = partitionRaftGroupName(
+                                                    ((ExtendedTableConfiguration) tableCfg).id().value(), i);
+
+                                            updateAssignmentsKeys(partId, baselineMgr.baselineNodes(), partCount, newReplicas,
+                                                    replicasCtx.storageRevision(), metaStorageMgr);
+                                        }
 
                                         return CompletableFuture.completedFuture(null);
```

Comment:
I think here we should return anything like [CompletableFuture.allOf](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletableFuture.html#allOf-java.util.concurrent.CompletableFuture...-) of all `updateAssignmentsKeys` futures.

Reply:
done